tests/spec/unit/podMod.spec.js

```diff
@@ -0,0 +1,107 @@
+var fs = require('fs'),
+	path = require('path'),
+	cordova = require(path.resolve(__dirname, '..', '..', '..', '..', 'cordova-lib/cordova-lib/src/cordova/cordova'));
```

Comment:
Found the issue. This reference will never work in a CI environment or testing in a single repo test. It probably worked in a dev environment where the cordova-lib repo was a sibling repo of cordova-ios.